tests/phpunit/class-wc-stripe-helper-test.php

```
@@ -295,6 +295,52 @@ public function provide_test_get_order_by_intent_id(): array {
+		} else {
+			$lookup_id = 'seti_unknown';
+		}
+
+		$result = WC_Stripe_Helper::get_order_by_setup_intent_id( $lookup_id );
+		if ( $expect_success ) {
+			$this->assertInstanceOf( WC_Order::class, $result );
+			$this->assertSame( $order->get_id(), $result->get_id() );
+		} else {
+			$this->assertFalse( $result );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_get_order_by_setup_intent_id is not named in camelCase. (undefined)

(CamelCaseMethodName)